src/grafana_dashboards/haproxy.json

```diff
@@ -12099,4 +12099,4 @@
   "uid": "rEqu1u5ue",
```

Comment:
out of scope: uid should be null or absent

Reply:
Got it, will update it in a follow-up PR